orbitx/flight_gui.py

```diff
@@ -125,6 +90,260 @@ def callback(*_):
         self.recentre_camera(DEFAULT_CENTRE)
     # end of __self__
 
+    ####################### pritvate methods to initial settings ############################
+
+    # def _vpython_version_check(self) -> None :
```

Comment:
thank you for removing this version check, can you also delete this commented out code?

Reply:
ok